chapters/research-design.tex

```diff
@@ -1,288 +1,586 @@
-%------------------------------------------------
+%-----------------------------------------------------------------------------------------------
 
 \begin{fullwidth}
 Research design is the process of structuring field work
 -- both experimental design and data collection --
 that will answer a specific research question.
 You don't need to be an expert in this,
 and there are lots of good resources out there
-that focus on designing interventions and evaluations.
-This section will present a very brief overview
-of the most common methods that are used,
-so that you can have an understanding of
-how to construct appropriate counterfactuals,
-data structures, and the corresponding code tools
-as you are setting up your data structure
-before going to the field to collect data.
-
-You can categorize most research questions into one of two main types.
-There are \textbf{cross-sectional}, descriptive, and observational analyses,
-which seek only to describe something for the first time,
-such as the structure or variation of a population.
-We will not describe these here,
-because there are endless possibilities
-and they tend to be sector-specific.
-For all sectors, however, there are also causal research questions,
-both experimental and quasi-experimental,
-which rely on establishing \textbf{exogenous variation} in some input
-to draw a conclusion about the impact of its effect
-on various outcomes of interest.
-We'll focus on these \textbf{causal designs}, since the literature
-offers a standardized set of approaches, with publications
-and code tools available to support your work.
+that focus on designing interventions and evaluations
+as well as on econometric approaches.
+This section will present a brief overview
```

Comment:
I think we need to make a stronger statement here about the value added of this chapter; we say just before that there are lots of good resources available, need to say more explicitly here how this is different.

Reply:
Tried again in 11b91f55827c40ba40a012f22d4478109808c867